Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Support FROM override
-ARG BUILD_IMAGE=docker.io/golang:1.20.7@sha256:741d6f9bcab778441efe05c8e4369d4f8ff56c9a635a97d77f55d8b0ec62f907
+ARG BUILD_IMAGE=docker.io/golang:1.20.8@sha256:1b46d1164aceec47750cc0c3a6f8c44ee8050a0745c07affcef73518544a2b1f
```

Comment:
This SHA is incorrect. `6b29720c5598cb40b23dd38f522050a698d427c3a5696c5efe1ed22bd45b1396` is correct SHA for golang:1.20.8 for amd64.

Reply:
I checked from here: https://hub.docker.com/layers/library/golang/1.20.8/images/sha256-1b46d1164aceec47750cc0c3a6f8c44ee8050a0745c07affcef73518544a2b1f?context=explore 
I will change it

Reply:
Yes its a bummer and pretty inconvenient that the digest on the hub doesn't match arch specific digest when you download, issue is still open as I can see https://github.com/docker/hub-feedback/issues/1925

Reply:
It is just misleading, not mismatching. Digest on the page is manifest digest, not image digest for architecture.
Easiest way to get correct one is to just pull it on a arm64 linux machine.

Reply:
Thats why I called mismatch. The hub has os/arch tab and the digest under the image changes when you switch os/arch. So I thought the digest was image digest. Only after you pointed out, I came to know that digest is actually manifest digest. Anyways arm64 or you meant amd64?

Reply:
I meant amd64 like I said. Arm64 would be incorrect for us.

Reply:
Ok, previous comment has arm64 so I got confused.

Reply:
Oh, I did not even notice the latter one :) Kept staring the original comment, looking for the typo.